src/main/kotlin/parts/wisdom/arcraider/visualization/GridPane.kt

```diff
@@ -0,0 +1,42 @@
+package parts.wisdom.arcraider.visualization
+
+import java.awt.Color
+import java.awt.Graphics
+import javax.swing.JPanel
+
+// When calculating the width / height of each grid pixel, offset it so
+// we don't fill right to the edge of the screen.
+private const val OFFSET = 10
+
+class GridPane(val grid: Grid) : JPanel() {
+    override fun paintComponent(g: Graphics) {
+        super.paintComponent(g)
+        background = Color.BLACK
+        val graphics2d = g.create()
+        graphics2d.color = Color.WHITE
+
+        val xSize = (width - OFFSET) /  grid.width
```

Comment:
I'm imagining that `width` refers to `this.getWidth()`? And that it is Kotlin exposing `getWidth()` from the `JPanel`? Might be worth a comment, whether I'm right or not. And is this different from the `getWidth()` just below, or the same? If they are the same, then of course express them the same way, else definitely worth another comment.

I had a really hard time understanding
```
xSize = (width - OFFSET) / grid.width
```
What would you think of using a more specific name than `xSize`, and then also suffixing many of the names with their units? E.g. 
```
val xPixelsPerSquare = (width - PANEL_INSET_PIXELS) / grid.widthSquares
```

Reply:
fixed